store/store.go

```diff
@@ -25,6 +26,8 @@ var (
 	statePrefix          = "s"
 	responsesPrefix      = "r"
 	metaPrefix           = "m"
+	hashToHeightPrefix   = "ha2h"
+	heightToHashPrefix   = "h2ha"
```

Comment:
We don't need this. First of all, you made `indexPrefix` unused.
But the main reason is that we need only one index - hash to height index. Keeping two of them defeats the purpose of this optimization (https://github.com/rollkit/rollkit/issues/1494 is about reducing storage usage).